db/migrate/20211221132609_add_membership_verify_token_to_people.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
-
-#  Copyright (c) 2012-2021, Schweizerischer Kanu-Verband. This file is part of
-#  hitobito_skv and licensed under the Affero General Public License version 3
-#  or later. See the COPYING file at the top-level directory or at
-#  https://github.com/hitobito/hitobito_skv.
-
-class AddMembershipVerifyTokenToPeople < ActiveRecord::Migration[6.1]
```

Comment:
Bitte die alte Migration nicht löschen, das führt nur zu Folgeproblemen. Falls es ein Problem ist bei neu aufgesetzten Entwicklungsumgebungen, dass zuerst die Core Migrations und erst dann alle Wagon Migrations angewendet werden, würde ich eher ein `if_not_exists: true` hier in diese alte Migration einfügen